src/jvm/test/resources/expected_logs/run_concurrent_test/array_rw.txt

```diff
@@ -0,0 +1,35 @@
+= Invalid execution results =
+| -------------------------------------- |
+|                Thread 1                |
+| -------------------------------------- |
+| run(() -> R): IllegalStateException #1 |
```

Comment:
In case of `runConcurrentTest`, we do not need to show the scenario. Instead, we can show the exception followed by the short and detailed interleavings.

Reply:
Done, please have a look at reports